python/ray/autoscaler/node_provider.py

```diff
@@ -118,7 +118,7 @@ def _import_external(provider_config):
 }
 
 
-def _load_class(path):
+def _load_class(path: str) -> Any:
```

Comment:
```suggestion
def _load_class(path: str) -> type:
```

Reply:
Not sure if this is the right place for this, but will ask anyway:
Can you explain how the annotation "type" differs from "Any"?